pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleJsonServlet.java

```diff
@@ -100,7 +102,11 @@ protected boolean isValidResource(@Nonnull Resource resource) {
         // Convert date string to UTC
         Date dateModified = new Date(resource.getResourceMetadata().getModificationTime());
         moduleMap.put("date_modified", dateModified.toInstant().toString());
-        moduleMap.put("body", releasedContent.get().cachedHtml.get().data.get());
+        // Return the body content of the module ONLY
+        moduleMap.put("body",
+                Html.parse(Charsets.UTF_8.name())
+                        .andThen(Html.getBody())
+                        .apply(releasedContent.get().cachedHtml.get().data.get()));
```

Comment:
I'm not sure how expensive, we would have to benchmark the jsoup parsers (and the modification). My thought was that this content **would** be cached by the front end, and so it would minimize the number of invocations. But we can optimize in the future if it ends up becoming a bottleneck.